openslides/core/views.py

```diff
@@ -35,6 +36,14 @@ class IndexView(utils_views.View):
     to the custom staticfiles directory. See STATICFILES_DIRS in settings.py.
     """
 
+    @classmethod
```

Comment:
This has to be a feature of this view class and not only of the one view in the urls